### DISCUSSION TITLE
Live de semana 2.

### DISCUSSION THREAD

**Discussion by Hafid-Davila** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#54] [Q&A]
Hola, no se si soy yo pero aún no veo el enlace del video de esta semana. ¿Alguien sabe si ya está disponible?

**Comment by Takzer94** [comment]
Ya, en la semana 2 ahí esta el link

**Comment by AlexAntonioG** [comment]
Esta disponible en Teams, en el apartado de semana 2, de todas formas te lo dejo [aquí](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZjVmMjA2MTAtZWNlYS00MTcxLTg4NTAtMjEzNTNlMzg3ODgy%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d) 😄

**Comment by armap99** [comment]
https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZjVmMjA2MTAtZWNlYS00MTcxLTg4NTAtMjEzNTNlMzg3ODgy%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d
